pkg/deploys/fuse/deployer.go

```diff
@@ -222,21 +220,13 @@ func (fd *FuseDeployer) createFuseCustomResource(namespace, brokerNamespace, use
 }
 
 // Get route hostname for fuse
-func (fd *FuseDeployer) getRouteHostname(namespace, brokerNamespace string, k8sclient kubernetes.Interface) (string, error) {
-	brokerDeployment, err := k8sclient.ExtensionsV1beta1().Deployments(brokerNamespace).Get("msb", metav1.GetOptions{})
-	if err != nil {
-		glog.Errorf("Failed to get managed services broker deployment: %+v", err)
-		return "", errors.Wrap(err, "failed to get managed services broker deployment")
+func (fd *FuseDeployer) getRouteHostname(namespace string) string {
+	routeHostname := namespace
+	routeSuffix, exists := os.LookupEnv("ROUTE_SUFFIX")
+	if exists {
+		routeHostname = routeHostname + "." + routeSuffix
 	}
-
-	for _, v := range brokerDeployment.Spec.Template.Spec.Containers[0].Env {
-		if v.Name == "ROUTE_SUFFIX" {
-			return namespace + "." + v.Value, nil
-		}
-	}
-
-	glog.Errorf("Failed to get cluster route subdomain from the managed services broker ROUTE_SUFFIX environment variable")
-	return "", errors.Wrap(err, "failed to get cluster route subdomain")
+	return routeHostname
 }
```

Comment:
@philbrookes @JameelB Was there any reason this was looking for the "ROUTE_SUFFIX" value directly on the "msb" deployment resource rather than reading the env var?

Reply:
No particular reason, I think at the time i just didn't think of it! Feel free to change it to just reading the env var :)